cider/function/FunctionLookupEngine.cpp

```diff
@@ -73,12 +67,11 @@ const SQLOps FunctionLookupEngine::getFunctionScalarOp(
             from_platform_));
   }
   const std::string& func_name = function_signature.func_name;
-  const auto& functionSignature =
-      cider::function::substrait::SubstraitFunctionSignature::of(
-          func_name, function_signature.arguments, function_signature.return_type);
-  const auto& functionOption =
-      scalar_function_look_up_ptr_->lookupFunction(functionSignature);
-  if (functionOption.has_value()) {
+  const std::vector<io::substrait::TypePtr>& arguments = function_signature.arguments;
+  const io::substrait::TypePtr& return_type = function_signature.return_type;
+  const auto& functionVariantPtr =
```

Comment:
functionVariantPtr --> function_variant_ptr and other local variables to keep variable naming consistent

Reply:
Done.